test/integration/red_test.go

```diff
@@ -222,6 +237,60 @@ func testSpanMetricsForHTTPLibrary(t *testing.T, svcName, svcNs string) {
 	})
 }
 
+// **IMPORTANT** Tests must first call -> func testREDMetricsForJSONRPCHTTP(t *testing.T, url, svcName, svcNs string) {
+func testSpanMetricsForJSONRPCHTTP(t *testing.T, svcName, svcNs string) {
+	pq := prom.Client{HostPort: prometheusHostPort}
+	var results []prom.Result
+
+	expectedSpanName := "Arith.M /jsonrpc"
+
+	// Test span metrics
+	test.Eventually(t, testTimeout, func(t require.TestingT) {
+		var err error
+		results, err = pq.Query(`traces_span_metrics_duration_count{` +
```

Comment:
So @titaneric , I managed to debug your test, and the problem lies on the name of this metric. Try this:
```suggestion
		results, err = pq.Query(`traces_span_metrics_duration_seconds_count{` +
```

Reply:
I also rebased your branch (locally of course) on top of the latest `main`. That and the comment above did the trick. Once you update your branch, I will retrigger the test and we should be good to go ;)

Reply:
For future reference, to debug the test, I manually launched the docker compose file, triggered a request with curl, and checked the results in the prometheus ui:
```
OTEL_EBPF_EXECUTABLE_PATH=testserver docker compose -f test/integration/docker-compose.yml up

curl -H "Content-Type: application/json" -d @body/formated.json  http://localhost:8088/jsonrpc

firefox http://localhost:9090
```

Reply:
Thank you for your advice, it's very helpful!